### PR TITLE
Add Search to Website

### PR DESCRIPTION
**Pull request by diberry**

### CODE REVIEW

**Comment by diberry** [comment]
@HeidiSteen @dereklegenzoff Please review. I need this PR merged before I can pull in the code to the docs. 


Here are my changes:
. Updated readmes as appropriate to align with new tutorial
. Moved configs into shared config file and added error if they are empty

Didn’t touch: 
. react app
. core functionality of API app

**Comment by diberry** [comment]
I fixed the screen shot - @dereklegenzoff  was right about that. Can one of you please merge? 

@HeidiSteen

**Comment by dereklegenzoff** [comment]
@diberry - merged! Updated extension should be published today too

**Comment by diberry** [comment]
@dereklegenzoff Thank you - that is awesome - what Version # should I tell the customer to look for?

File: search-website/README.md [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+This project can be run anywhere, but VS Code is required for local debugging.
+
+1. Open the `./search-website` application with VS Code.
+
+### Running the front-end
+
+1. From a terminal in the `./search-website` root, install front-end dependencies...
+
+   ```bash
+   npm install
```
Comment (HeidiSteen) [inline search-website/README.md:62]: I think this step requires path information.  I ran it, but it says "no such file or directory, open 'C:\Users\heidist\package.json'"   For testing, I put the folder off my C root drive, not my users drive.  I'll continue testing, but I think i'm probably not in good shape at this point.

File: search-website/README.md [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+1. From a different terminal in the `./search-website/api` root, install front-end dependencies...
+
+   ```bash
+   npm install
+   ```
+
+1. Run the api project.
+
+   ```bash
+   npm start
```
Comment (HeidiSteen) [inline search-website/README.md:82]: This failed for me: 
PS C:\search-website\api> npm install
npm WARN api@1.0.0 No repository field.
npm WARN api@1.0.0 No license field.

added 38 packages from 134 contributors and audited 38 packages in 1.746s
found 0 vulnerabilities

PS C:\search-website\api> npm start

> api@1.0.0 start C:\search-website\api
> func start

'func' is not recognized as an internal or external command,
operable program or batch file.
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! api@1.0.0 start: `func start`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the api@1.0.0 start script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\heidist\AppData\Roaming\npm-cache\_logs\2021-03-16T17_07_29_275Z-debug.log
PS C:\search-website\api>

**Review by HeidiSteen — COMMENTED** [review]
I got as far as I could before running into a blocking error.  I'll go look at the tutorial to see if the screenshots help me, but in the mean time, I do have feedback on the readme.

File: search-website/README.md (hunk 2 above)
Comment (diberry) [reply search-website/README.md:82]: I'll update the dependencies to include Azure Function Core tools: 

```bash
npm install -g azure-functions-core-tools
```

**Comment by dereklegenzoff** [comment]
@diberry - version will be 0.2.0

**Pull request merged**